lms/djangoapps/course_goals/api.py

```diff
@@ -28,6 +29,27 @@ def add_course_goal(user, course_id, goal_key):
     )
 
 
+def add_course_goal(user, course_id, number_of_days_with_visits_per_week=None,
+                    subscribed_to_goal_reminders=None):
```

Comment:
Why have the `None` defaults? `number_of_days` isn't nullable right? That would result in an error? And you always want to be passed both arguments. I'd say just make them args, not kwargs.

Reply:
good idea, i had them as kwargs before your suggestion to make separate methods, making them args now makes sense